src/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -225,26 +236,26 @@ HandlerResolver::FindNativeAssemblyFromHostfxr(
         }
     }
 
-    RETURN_IF_FAILED(struNativeSearchPaths.SyncWithBuffer());
+    struNativeSearchPaths.resize(struNativeSearchPaths.find(L'\0'));
```

Comment:
Seems to be repeated operations between this line and the one prior.

Reply:
`            struNativeSearchPaths.resize(dwBufferSize);` only kicks in when there is enough space. In successfull case dwRequiredBufferSize would be 0 unfortunately.

Reply:
I see.